nuage-tripleo-heat-templates/scripts/pull_nuage_containers/nuage_overcloud_images.yaml.j2

```diff
@@ -0,0 +1,17 @@
+parameter_defaults:
+  DockerHeatApiCfnConfigImage: {{ nuage_config['local_registry'] }}/nuagenetworks/rhosp{{ nuage_config['version'] }}-openstack-heat-api-cfn-{{ nuage_config['release'] }}:{{ nuage_config['tag'] }}
+  DockerHeatApiCfnImage: {{ nuage_config['local_registry'] }}/nuagenetworks/rhosp{{ nuage_config['version'] }}-openstack-heat-api-cfn-{{ nuage_config['release'] }}:{{ nuage_config['tag'] }}
+  DockerHeatApiConfigImage: {{ nuage_config['local_registry'] }}/nuagenetworks/rhosp{{ nuage_config['version'] }}-openstack-heat-api-{{ nuage_config['release'] }}:{{ nuage_config['tag'] }}
+  DockerHeatApiImage: {{ nuage_config['local_registry'] }}/nuagenetworks/rhosp{{ nuage_config['version'] }}-openstack-heat-api-{{ nuage_config['release'] }}:{{ nuage_config['tag'] }}
+  DockerHeatConfigImage: {{ nuage_config['local_registry'] }}/nuagenetworks/rhosp{{ nuage_config['version'] }}-openstack-heat-api-{{ nuage_config['release'] }}:{{ nuage_config['tag'] }}
+  DockerHeatEngineImage: {{ nuage_config['local_registry'] }}/nuagenetworks/rhosp{{ nuage_config['version'] }}-openstack-heat-engine-{{ nuage_config['release'] }}:{{ nuage_config['tag'] }}
+  DockerHorizonConfigImage: {{ nuage_config['local_registry'] }}/nuagenetworks/rhosp{{ nuage_config['version'] }}-openstack-horizon-{{ nuage_config['release'] }}:{{ nuage_config['tag'] }}
+  DockerHorizonImage: {{ nuage_config['local_registry'] }}/nuagenetworks/rhosp{{ nuage_config['version'] }}-openstack-horizon-{{ nuage_config['release'] }}:{{ nuage_config['tag'] }}
+  DockerInsecureRegistryAddress:
+  - {{ nuage_config['local_registry'] }}
+  DockerNeutronApiImage: {{ nuage_config['local_registry'] }}/nuagenetworks/rhosp{{ nuage_config['version'] }}-openstack-neutron-server-{{ nuage_config['release'] }}:{{ nuage_config['tag'] }}
+  DockerNeutronConfigImage: {{ nuage_config['local_registry'] }}/nuagenetworks/rhosp{{ nuage_config['version'] }}-openstack-neutron-server-{{ nuage_config['release'] }}:{{ nuage_config['tag'] }}
+  {% if 'nova-compute' in nuage_config['nuage_images' %}
```

Comment:
its ok to have extra images in this env file. Unless docker/service/nova-compute-avrs.yaml is called in a deployment, these won't come into picture at all. So, we can remove this condition.

Reply:
Error: jinja2.exceptions.TemplateSyntaxError: unexpected '}', expected ']'
Add ']' after "/'nuage_images/'"